resources/css/bem/contest-voting-list.less

```diff
@@ -125,6 +125,13 @@
     }
   }
 
+  &__title-link {
+    .link-white();
+    .u-relative();
```

Comment:
don't include utility class like this; use normal css instead

Reply:
Is it a code style thing? Just curious because I"m seeing other files use utility functions just like this

Reply:
yes those are mistakes

Reply:
Is it ok to use it in an element className?

Reply:
as in the js? well yeah

Reply:
utility classes are the ones with `u-` prefix...

Reply:
oh so the link-white is fine to use?

Reply:
well yes, it's a function, not a class